### PR TITLE
net/frr: add IPv6 to BGP

### PR DESCRIPTION
**Pull request by mimugmail**
- changed masks and field types
- extended prefix lists
- restructured template to new address-families

### CODE REVIEW

File: net/frr/src/opnsense/mvc/app/models/OPNsense/Quagga/BGP.xml [hunk 1]
```diff
@@ -37,10 +37,9 @@
                                 <default>1</default>
                                 <Required>Y</Required>
                         </enabled>
-                        <address type="TextField">
+                        <address type="HostnameField">
```
Comment (fabianfrz) [inline net/frr/src/opnsense/mvc/app/models/OPNsense/Quagga/BGP.xml]: should this support hostnames too?

Comment (mimugmail) [reply net/frr/src/opnsense/mvc/app/models/OPNsense/Quagga/BGP.xml]: Normally not, but HostnameField is ATM the only fieldtype validating v6 addresses. Didn't found a good mask to validate v4 and v6 so I asked @AdSchellevis in another project for this type. Don't know if he also likes to introduce a new fieldtype for v4 and v6 only, or you have an idea for stable masking v4 and v6?

Comment (fabianfrz) [reply net/frr/src/opnsense/mvc/app/models/OPNsense/Quagga/BGP.xml]: How about NetworkField?
That is v4 and v6 + optional CIDR

**Comment by mimugmail** [comment]
Wie konnte ich denn NetworkField vergessen .. boah .. sorry!

File: net/frr/src/opnsense/mvc/app/models/OPNsense/Quagga/BGP.xml [hunk 2]
```diff
@@ -1,6 +1,7 @@
 <model>
     <mount>//OPNsense/quagga/bgp</mount>
     <description>BGP Routing configuration</description>
+    <version>1.1.0</version>
```
Comment (fichtner) [inline net/frr/src/opnsense/mvc/app/models/OPNsense/Quagga/BGP.xml]: undefined is 0.0.0, so we would better start with 1.0.0 here?

**Comment by mimugmail** [comment]
Done, ready for merge :)

**Comment by fichtner** [comment]
Merged, thanks!

**Pull request merged**